### DISCUSSION TITLE
Using prisma to migrate but running into unexpected error

### DISCUSSION THREAD

**Discussion by nicholasraphael** [electric-sql/electric#939] [Questions]
I'm new to using electric and trying to work with prisma but keep getting the following trying to use migrations `Error: unexpected message from server
   0: sql_schema_connector::flavour::postgres::connection::describe_schema
           with namespaces=None
             at schema-engine/connectors/sql-schema-connector/src/flavour/postgres/connection.rs:75
   1: sql_schema_connector::validate_migrations
           with namespaces=None
             at schema-engine/connectors/sql-schema-connector/src/lib.rs:324
   2: schema_core::state::DevDiagnostic`

any idea why.

Here's the schema 
```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model User {
  @@map("users")

  id String @id @default(cuid())
  email String @unique
}
```

and my env `DATABASE_URL="postgresql://postgres:proxy_password@localhost:65432/postgres"`

Thanks

**Comment by alco** [comment]
@nicholasraphael See if you can still reproduce the issue using the latest `electricsql/electric:canary` image. We've just merged a fix for the Electric proxy that could have addressed this - https://github.com/electric-sql/electric/pull/998.
